### PR TITLE
Update the EBL directory

### PR DESCRIPTION
**Pull request by bkhelifi**
This PR updates the references to the models, as well as the figure

### CODE REVIEW

**Review by maxnoe — CHANGES_REQUESTED** [review]
There is a typo in the legend for Francesch__i__ni 2017

**Review by maxnoe — APPROVED** [review]

**Comment by bkhelifi** [comment]
Thanks @maxnoe. I am still waiting for the feedback from Finke, Dominguez ans Saldana-Lopez about the licence question.
I propose that we merge this PR once we have all the authors' information.

**Comment by bkhelifi** [comment]
Add the licences are added. This PR is ready to be reviewed... @registerrier

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi . Looks good to me.

File: ebl/README.md
Comment (registerrier) [inline ebl/README.md:7]: A question. Is it just the first author here?

File: ebl/README.md
Comment (registerrier) [inline ebl/README.md:21]: Same.

**Review by registerrier — APPROVED** [review]
Thanks @bkhelifi !

**Pull request merged**
